### PR TITLE
Docs persistence

### PR DESCRIPTION
**Pull request by mlejva**
Improve code snippets

### CODE REVIEW

File: apps/web/src/app/(docs)/docs/sandbox/persistence/page.mdx [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 <Note>
 Sandbox persistence is currently in beta:
-1. [Reach out to us](/docs/support) to get access to the beta.
-1. You need to install the [beta version of the SDKs](#installing-the-beta-version-of-the-sdks).
+1. [Reach out to us](/docs/support) with your use caseto get access to the beta.
```
Comment (0div) [inline apps/web/src/app/(docs)/docs/sandbox/persistence/page.mdx]:
```suggestion
1. [Reach out to us](/docs/support) with your use case to get access to the beta.
```

File: apps/web/src/app/(docs)/docs/sandbox/persistence/page.mdx [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
+1. You'll need to install the [beta version of the SDKs](#installing-the-beta-version-of-the-sdks).
```
Comment (0div) [inline apps/web/src/app/(docs)/docs/sandbox/persistence/page.mdx:6]: bullet points numbering are all 1s

Comment (mlejva) [reply apps/web/src/app/(docs)/docs/sandbox/persistence/page.mdx:6]: That's okay. Markdown handles that for you automatically. This way, you don't need to change numbers every time you add a new item.

https://e2b-fp29dw6dg-e2b.vercel.app/docs/sandbox/persistence

Screenshot 2024-12-17 at 12 54 23 PM

Comment (0div) [reply apps/web/src/app/(docs)/docs/sandbox/persistence/page.mdx:6]: cool, TIL

**Pull request merged**